contrib/spec/podman.spec.in

```diff
@@ -535,6 +535,7 @@ export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
 %{_usr}/lib/systemd/user/podman.socket
 %{_usr}/lib/systemd/user/podman-auto-update.service
 %{_usr}/lib/systemd/user/podman-auto-update.timer
+%{_usr}/lib/systemd/user/podman-restart.sevice
```

Comment:
Typo ("sevice")

Reply:
Assuming you run Fedora on your laptop, or otherwise have access to `rpmbuild`, these are the steps run by the 'Make RPM' step in CI:
https://github.com/containers/podman/blob/763abaea36ad289a887a0b61d52dcc8d3bea84d8/contrib/cirrus/runner.sh#L227-L229